docs/configuring-playbook-backup-borg.md

```diff
@@ -0,0 +1,52 @@
+# Setting up borg backup (optional)
+
+The playbook can install and configure [borgbackup](https://www.borgbackup.org/) with [borgmatic](https://torsion.org/borgmatic/) for you.
+The backup will run based on `matrix_backup_borg_schedule` var (systemd timer calendar), default: 4am every day
+
+## Prerequisites
+
+1. Create ssh key:
+
+```bash
+ssh-keygen -t ed25519 -N '' -C matrix
```

Comment:
Maybe it should be pointed out that you can run this command on any machine, not necessarily on the Matrix server.

Adding `-f matrix-borg-backup` may make sense. Otherwise, someone might override their existing `~/.ssh/id_ed25519{,.pub}` files.